libs/emblem/src/components/emblem-base.tsx

```diff
@@ -1,14 +1,14 @@
 import { useState, type ImgHTMLAttributes } from 'react';
 import { FALLBACK_URL } from '../config/index';
 
-export type ImgProps = ImgHTMLAttributes<HTMLImageElement>;
+export type ImgProps = ImgHTMLAttributes<HTMLImageElement> & { size?: number };
```

Comment:
**thought (non-blocking)**:  having `ImgHTMLAttributes` be the base here makes.. increasingly little sense. I'll make a refactor ticket to get right of this, outside this pr.